test/built-ins/Temporal/Calendar/prototype/prop-desc.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2020 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal-calendar-objects
+description: The "prototype" property of Temporal.Calendar
+includes: [propertyHelper.js]
+features: [Temporal]
+---*/
+
+const { Calendar } = Temporal;
```

Comment:
It's probably my fault for introducing this pattern in the first place, but I'd prefer not to propagate it, and just use `Temporal.Calendar` below.

Reply:
Done!